src/main/java/Parser.java

```diff
@@ -126,8 +126,8 @@ public String parse(String nextCommand, TaskList taskList, Storage storage, Scan
             outputString.append("    ____________________________________________________________\n");
```

Comment:
Perhaps splitting the separator string into 2 lines using a "+" would be neater according to coding style guidelines?

Reply:
I decided to use StringBuilder so like concatenating strings with the + operation would be less efficient than doing a .append. Or at least I think that's how it's supposed to work?